include/deal.II/base/geometry_info.h

```diff
@@ -1729,6 +1729,7 @@ struct GeometryInfo
    * face_to_cell_vertices() function.
    */
   static
+  inline
```

Comment:
Why did you need to put the `inline` statement at the place of the declaration? Our convention in deal.II is to not put the `inline` here (except for `DEAL_II_ALWAYS_INLINE`). Actually, for templated classes/functions it is not even needed at the place of the definition - even though we still use that convention.